layouts/index.html

```diff
@@ -340,16 +347,26 @@ <h2>Open Source</h2>
             <p>RAPIDS would not be possible without the collaboration of these important open source projects. Click on a logo to
               learn more: </p>
             <div class="d-flex flex-wrap justify-content-between mt-5">
-              {{ $row := slice "apache-arrow" "dask" "nuclio" "numba" "scikit-learn" "xgboost" }}
+              {{ $row := slice "apache-arrow" "dask" "nuclio" "networkx" "numba" "scikit-learn" "xgboost" }}
```

Comment:
These are alphabetical. Let’s match the existing style.
```suggestion
              {{ $row := slice "apache-arrow" "dask" "networkx" "nuclio" "numba" "scikit-learn" "xgboost" }}
```